content/pt/docs/concepts/instrumentation/libraries.md

```diff
@@ -4,9 +4,7 @@ description: Aprenda como adicionar instrumentação nativa à sua biblioteca.
 weight: 40
 default_lang_commit: a570a00c7a238ffe26528d7bfb20efdbaf939c39
 htmltest:
-  IgnoreDirs:
-    # TODO drop next line after https://github.com/open-telemetry/opentelemetry.io/issues/5423 is fixed for pt pages:
-    - ^pt/docs/concepts/instrumentation/libraries/
+  IgnoreDirs: {}
```

Comment:
I think you can drop this attribute if it is empty, cc @chalin

Reply:
Yes. Drop the `htmltest` entry entirely.

Reply:
There's a broader issue to be addressed though. I'll comment separately from this thread.